src/main/java/uk/gov/companieshouse/filingmock/processor/strategy/AcceptanceStrategyFactory.java

```diff
@@ -22,6 +23,8 @@ public static AcceptanceStrategy getStrategy(Transaction submission) {
             // There are multiple insolvency types e.g. insolvency#600 but all will start with "insolvency," and should use the same strategy
         } else if (submissionType.contains("insolvency")) {
             return INSOLVENCY;
+        } else if (submissionType.contains("cessation")) {
+            return CESSATION;
```

Comment:
We'll need test coverage for this in `AcceptanceStrategyFactoryTest`

Reply:
Changes applied